DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/main.cpp

```diff
@@ -78,6 +99,9 @@ int main(int argc, char *argv[]) {
 #ifdef FPGA_EMULATOR
   IndexT count = 128;
   int runs = 2;
+#elif FPGA_SIMULATOR
```

Comment:
How long does the simulation takes (roughly) seeing that is uses the same parameters as the emulation target?

Reply:
I don't know if the hostpipe simulation hang is fixed, but right now it's hanging. I would like to ask if I should add a timeout for simulation.